db/smt-db

```diff
@@ -304,7 +305,7 @@ sub setup
     my ($login,$pass,$uid,$gid) = getpwnam("wwwrun");
     chown 0, $gid, "/etc/smt.conf";
 
-    `smt-sql --select-mode-direct /usr/share/schemas/smt/mysql/2.03/100-smt-tables.sql > /var/log/smt/smt-db-setup.log`;
+    `smt-sql --select-mode-direct /usr/share/schemas/smt/mysql/3.00/100-smt-tables.sql > /var/log/smt/smt-db-setup.log`;
```

Comment:
smt11 used DBIx as well but it is horrible. I had massive trouble to write correct functions which works with this module.
The new way is a copy from SUSE Manager. SUSE Manager also has some kind of "latest" or "current" schema plus the upgrades.
I think nobody want to install with an old schema version which does not match the code.